packaging/windows/installer.nsi

```diff
@@ -265,14 +296,11 @@ FunctionEnd
 # FunctionEnd
 
 # --- Installation sections ---
-!define PROGRAM_UNINST_KEY "Software\Microsoft\Windows\CurrentVersion\Uninstall\${PROGRAM_NAME}"
-!define PROGRAM_UNINST_ROOT_KEY "HKLM"
-!define PROGRAM_UNINST_FILENAME "$INSTDIR\uninstall.exe"
 
 BrandingText "${PROGRAM_NAME} Windows Installer v${INSTALLER_SCRIPT_VERSION}"
 Name "${PROGRAM_NAME} ${PROGRAM_VERSION}"
 OutFile "${BUILD_DIR}\${INSTALLER_FILENAME}"
-InstallDir "$PROGRAMFILES\Parsec Cloud"
+InstallDir "$PROGRAMFILES64\${PROGRAM_NAME}\${PROGRAM_VERSION}"
```

Comment:
The previous uninstaller take care of removing the parsec files under `$PROGRAMFILES` ?

Reply:
Exactly

Reply:
:+1: